config/schedule.rb

```diff
@@ -167,6 +167,12 @@
     at: '4:02am',
     interruptable: true,
   },
+  {
+    # No-op for environments that don't have (1) the HMIS enabled, and (2) an "MPER" credential present
+    task: 'driver:hmis_external_apis:ac_hmis:update_unit_availability',
+    frequency: 3.minutes,
+    trigger: ENV['HMIS_OKTA_CLIENT_ID'] == 'true' && ENV['RAILS_ENV'] == 'production',
+  },
```

Comment:
After discussing with @gigxz we think this should actually be done as a delayed job added with a  +1 minute runtime to the DJ queue.  Happy to discuss @ttoomey

Reply:
Hi @eanders sounds good, thanks. Just ping me on slack when you have a few minutes to discuss